src/main/resources/interface-specification.yml

```diff
@@ -1262,6 +1262,8 @@ components:
       properties:
         prettyName:
           type: string
+        checksum:
```

Comment:
this is the temporary change we were talking about?
if yes, may we add a comment to remind us that it needs to be deleted (and not moved in the migration)? something like
```
# Temporarily required for feature PIN-4267, to be removed
```